tests/functional/heketi/test_block_volumes_heketi.py

```diff
@@ -371,25 +371,27 @@ def test_create_max_num_blockhostingvolumes(self):
                 self.heketi_client_node, self.heketi_server_url, bv_volume)
 
         # Check if any blockvolume exist in heketi & gluster
-        for bhv_volume in new_bhv_list[:]:
+        for bhv_id, bhv_name in new_bhv_list[:]:
             heketi_vol_info = heketi_volume_info(
                 self.heketi_client_node, self.heketi_server_url,
-                bhv_volume, json=True)
+                bhv_id, json=True)
             self.assertNotIn(
                 "blockvolume", heketi_vol_info["blockinfo"].keys())
             gluster_vol_info = get_block_list(
-                'auto_get_gluster_endpoint', volname="vol_%s" % bhv_volume)
+                'auto_get_gluster_endpoint', bhv_name)
             self.assertIsNotNone(
-                gluster_vol_info, "Failed to get volume info %s" % bhv_volume)
-            new_bhv_list.remove(bhv_volume)
+                gluster_vol_info, "Failed to get volume info %s" % bhv_name)
```

Comment:
```suggestion
                gluster_vol_info, "Failed to get volume info {}".format(bhv_name))
```

Reply:
These are old changes, which this file follows if we want to change those, so we need to change the remaining.
So Continuing with the old string format

Reply:
So does that mean we should add the new change in the same format if we want to add a new string?

Reply:
@adityaramteke check commit messsage, this change is not for fixing old string format. It's for fixing known bug.

Reply:
@adityaramteke +1 to check commit msg

Reply:
Okay, resolving this threading. Other changes are LGTM.